bridges/snowbridge/pallets/inbound-queue/src/lib.rs

```diff
@@ -321,17 +321,6 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
-		pub fn do_convert(
-			message_id: H256,
-			message: inbound::VersionedMessage,
-		) -> Result<(Xcm<()>, BalanceOf<T>), Error<T>> {
-			let (mut xcm, fee) =
-				T::MessageConverter::convert(message).map_err(|e| Error::<T>::ConvertMessage(e))?;
-			// Append the message id as an XCM topic
-			xcm.inner_mut().extend(vec![SetTopic(message_id.into())]);
```

Comment:
Was the message ID not set here anyway? 🤔

Reply:
Not for the inner xcm that is sent to the destination chain.

Reply:
Ahhh I see. Should it not be set on both the inner and outer then?

Reply:
Yes it is now being set, both inner and outer, inside the message converter.
Inner: https://github.com/Snowfork/polkadot-sdk/pull/134/files#diff-5a39b1d9ef60342361183a0aed74e5dae96dad6a973162b59c593705a31e3314R284
Outer: https://github.com/Snowfork/polkadot-sdk/pull/134/files#diff-5a39b1d9ef60342361183a0aed74e5dae96dad6a973162b59c593705a31e3314R299

Because I had to plumb the message_id to the converter to set the inner topic, I removed it from here and set the outer topic in the converter as well instead of doing it in two places(pallet and converter).

Reply:
I see, thanks!